component/attr_cache/attr_cache.go

```diff
@@ -119,6 +120,16 @@ func (ac *AttrCache) Stop() error {
 	return nil
 }
 
+// GenConfig : Generate the default config for the component
+func (ac *AttrCache) GenConfig() error {
+	log.Info("AttrCache::Configure : config generation started")
```

Comment:
I think command level (means in gen-Config command file) logging is enough.
Adding info level into every component, doesn't have any significance.

Reply:
Yes Ik anyways we won't get the logs as we aren't giving user to add the log path. I just added this log in case there is some error which leads to this method getting called in future without gen-config? Then we would know something's wrong. Idk I felt it's just a way to track incase of some issue in our code later on.